TrafficReplayer/src/org/opensearch/migrations/replay/TrafficReplayer.java

```diff
@@ -87,19 +89,22 @@ public static void main(String[] args) throws IOException, InterruptedException
         }
 
         var tr = new TrafficReplayer(uri);
-        try (var tripleWriter = new RequestResponseResponseTriple.TripleToFileWriter(outputPath)) {
-            ReplayEngine replayEngine = new ReplayEngine(
-                    rp -> tr.writeToSocketAndClose(rp,
-                            triple -> {
-                                try {
-                                    tripleWriter.writeJSON(triple);
-                                } catch (IOException e) {
-                                    System.err.println("Caught an IOException while writing triples to file.");
-                                    e.printStackTrace();
-                                }
-                            })
-            );
-            tr.runReplay(directoryPath, replayEngine);
+        try (var fileTriplesStream = new FileOutputStream(outputPath.toFile(), true)) {
+            try (var bufferedTriplesStream = new BufferedOutputStream(fileTriplesStream)) {
```

Comment:
Just FYI - you can also do 
`try (var fileTriplesStream = new FileOutputStream(outputPath.toFile(), true);
           var bufferedTriplesStream = new BufferedOutputStream(fileTriplesStream)) {`

Reply:
Ah, that's nicer. Thanks!